### DISCUSSION TITLE
How to create a light trail - Comparing brightest parts of two images

### DISCUSSION THREAD

**Discussion by shfury** [ImageMagick/ImageMagick#3180] [Help] [answered]
Hi!

I've got a tiff sequence of a light moving around in a dark room, the goal is to write out a new sequence in which the light forms a trail.

I'm thinking something like - the current frame and the most recent new frame are compared, whichever pixel is brighter gets put in the next new frame.

Is this possible with ImageMagick? Can anyone point me towards some relevant reading, or even better suggest some commands to use?

Thanks for reading

**Comment by snibgo** [comment, marked as answer]
Assuming you have input files named in_*.tiff, and these can all fit in memory at the same time, this makes a single output where each pixel has values set to the maximum of the corresponding pixels of the input files:
```
magick in_*.tiff -evaluate-sequence Max out.tiff
```

EDIT: Ah, no, sorry, I realise you want a sequence. For that, I would do it in a shell loop that calls magick for each pair of inputs.

That would be inefficient as each input is read twice.

**Reply by shfury** [reply]
Ok thank you, I'll have a go with this
